crypto/core_fetch.c

```diff
@@ -74,6 +75,7 @@ static void ossl_method_construct_this(OSSL_PROVIDER *provider,
                    algo->algorithm_name, algo->property_definition,
```

Comment:
I wonder why you are not looking at the return value of put ?

Reply:
Read the comment at (new) lines 53-62

Reply:
Ah, okay, maybe explicitly casting to void would make sense then.

Reply:
Done